src/Util/DigitizeUtil/DigitizeUtil.js

```diff
@@ -0,0 +1,81 @@
+import OlObservable from 'ol/observable';
+
+/**
+ * This class provides some static methods which might be helpful when working
+ * with redlining functions.
+ *
+ * @class DigitizeUtil
+ */
+class DigitizeUtil {
+
+  /**
+   * Moves / translates an `OlFeature` to the given `pixel` delta in
+   * in the end with given `duration` in ms, using the given style,
+   * calling a `doneFn`.
+   *
+   * @param {OlMap} map An OlMap.
+   * @param {ol.Feature} featureToMove The feature to move.
+   * @param {Number} duration The duration in ms for the moving to complete.
+   * @param {Array<Number>} pixel Delta of pixels to move the feature.
+   * @param {ol.style.Style} style The style to use when moving the feature.
+   * @param {Function} doneFn The function to call when done.
+   *
+   * @return {String} A listener key from a postcompose event.
+   */
+   static moveFeature = (map, featureToMove, duration, pixel, style, doneFn) => {
```

Comment:
I'd prefer to return a `Promise` instead of passing a callback function.

Reply:
Changed in 0504b8e